src/components/attendee/NavOverlay/NavOverlay.module.scss

```diff
@@ -107,6 +107,8 @@ $-z-index-base: 1;
   border-radius: border.$radius--xs;
   padding: 0;
   font-size: font.$size--xl;
+  // removes blue highlighting on mobile tap
```

Comment:
[Q] Is it only Webkit specific problem?

Reply:
yes, didn't really face the same problem in firefox